File: prime_number/prime.py
import math

def prime():
    num = int(input("Enter a number and we'll tell you if it's prime: "))
    sqrt = int(math.sqrt(num))
    for i in range(num - 1, sqrt, -1):
        if num % i == 0:
            print(f"the number [{num}] isn't prime")
            return
    print(f"the number [{num}] is prime")

prime()

File: prime_number/prime.c
#include <stdio.h>
#include <math.h>

void prime_number(){
    int num = 0;
    
    printf("Enter a number and we'll tell you if it's prime: ");
    scanf("%d", &num);
    
    for (int i = num -1; i > sqrt(num); i--){
    	if(num % i == 0){
    		printf("the number [%d] isn't prime", num);
    		return;
		}
	}
    printf("the number [%d] is prime", num);
}


int main(){
	
	prime_number();
	
    return 0;
}
